### PR TITLE
Fix URL for downloading keyboard list on Windows version

### PR DESCRIPTION
**Pull request by drashna**

### CODE REVIEW

**Comment by skullydazed** [comment]
It's actually line 238 that needs to be changed.

`var json = wc.DownloadString("https://compile.qmk.fm/v1/keyboards");`

should be

`var json = wc.DownloadString("http://api.qmk.fm/v1/keyboards");`

**Comment by shelaf** [comment]
I think
```
var json = wc.DownloadString("https://api.qmk.fm/v1/keyboards");
```
is better.

File: windows/QMK Toolbox/MainWindow.cs [hunk 1]
```diff
@@ -235,7 +235,7 @@ private void LoadKeyboardList()
             {
                 using (var wc = new WebClient())
                 {
-                    var json = wc.DownloadString("https://compile.qmk.fm/v1/keyboards");
+                    var json = wc.DownloadString("http://api.qmk.fm/v1/keyboards");
```
Comment (fauxpark) [inline windows/QMK Toolbox/MainWindow.cs]: Why is this HTTP now?

Comment (drashna) [reply windows/QMK Toolbox/MainWindow.cs]: because that's what skully said/linked

Comment (yanfali) [reply windows/QMK Toolbox/MainWindow.cs]: It's so we don't trigger the cert issues, due to the super old version of .NET

**Comment by jackhumbert** [comment]
Thanks :)

**Comment by parkerault** [comment]
I know this has already been fixed, but for users of older versions this could be solved by setting up a DNS redirect to the new url.

**Pull request merged**